### PR TITLE
Rewrite of Toon component

### PR DESCRIPTION
**Pull request by frenck**
**Description:**

Updates documentation to match the rewrite of the component in the parent PR.

Personal reference: `frenck-2019-0027`

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21186

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/toon.markdown [hunk 1]
```diff
@@ -77,3 +77,7 @@ The `toon` climate platform allows you to interact with your Toon thermostat. Fo
 | Heat           | Thuis   |
 | Eco            | Weg     |
 | Cool           | Slapen  |
+
+It also supports setting the temperature manually.
+
+The Toon API is polled at a 30-second interval, so the status is relatively fresh without overloading the API.
```
Comment (scheric) [inline source/_components/toon.markdown:83]:
```suggestion
<p class='note'>
You can not use the google assistant for changing the temperature.

The Toon API is polled at a 30-second interval, so the status is relatively fresh without overloading the API.
</p>
```

**Comment by scheric** [comment]
Should we mention that you can't control the thermostat with the google assistant?

**Comment by klaasnicolaas** [comment]
@scheric why? What is de direct relation between Toon and google assistant?

**Comment by scheric** [comment]
I would think that you can use the Google assistant with most of the the climate devices in home assistant. `I have not tested this` 

You can not use the Google assistant `possible others too` for this entity because there needs to be an extra state in home assistant if you set an custom themperature via the Google assistant.

**Comment by csp1980** [comment]
when is this going to be released in hassio?
i am running hassio 88.1 and hassos 2.10

i have tried to set it up in the config.yaml like this
toon:
  client_id: my consumer key
  client_secret: my consumer secret

when i do a config check it gifs an error

**Comment by scheric** [comment]
@csp1980 
This is not the place to look if it is implemented. look at  [home-assistant/home-assistant#21186](https://github.com/home-assistant/home-assistant/pull/21186)
It could be implemented in the next version.

### note
if you are using that layout it will not work. Use this.
```yaml
toon:
  client_id: [key]
  client_secret: [secret]
```

File: source/_components/toon.markdown [hunk 2]
```diff
@@ -21,52 +21,52 @@ redirect_from:
...
 - [Climate](#climate)
 - Sensor
-- Switch
 
-For the `toon` component to work, you'll need an active Toon subscription with Eneco. The component uses your Mijn Eneco credentials to control your thermostat through the [toonopafstand](https://toonopafstand.eneco.nl) domain.
+## {% linkable_title Setting up a developer account %}
+
+In order to be able to use this component, you'll need to sign up for a free Toon API developer account.
+
+1. Visit the [Toon API developers website](https://developer.toon.eu/), and [sign in](https://developer.toon.eu/user/login). [Create an account](https://developer.toon.eu/user/register) if you don’t have one already.
```
Comment (scheric) [inline source/_components/toon.markdown:38]:
```suggestion
1. Visit the [Toon API developers website](https://developer.toon.eu/), and [sign in](https://developer.toon.eu/user/login). [Create an account](https://developer.toon.eu/user/register) if you don’t have one already. These credentials do not have to be the same as your Eneco account.
```

**Comment by csp1980** [comment]
I all ready done that. But hassio says client_id is an invalid option for toon

**Comment by scheric** [comment]
@csp1980
That is because it is not jet implemented. Look at my other post!

File: source/_components/toon.markdown [hunk 3]
```diff
@@ -21,52 +21,52 @@ redirect_from:
   - /components/switch.toon/
 ---
 
-The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat and sensors for power and gas consumption. The component also auto-detects any smart plugs, solar panels and smoke detectors connected to Toon and adds sensors and switches for them.
+The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat, sensors for power and gas consumption, sensors for solar production and several binary sensors for things like boiler burner on/off, hot tap water and boiler health status.
```
Comment (scheric) [inline source/_components/toon.markdown:24]:
```suggestion
The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat, sensors for power and gas consumption, sensors for solar production, several binary sensors for things like boiler burner on/off, hot tap water and boiler health status.
```

**Review by scheric — APPROVED** [review]
At the state it is now it is sufficient for users who want to use this component.

***
Minor changes can be made but are not necessary for understanding what you can do with it.

**Comment by frenck** [comment]
Parent PR closed for now, will re-open when ready.

**Comment by frenck** [comment]
Had some time to provide the unit tests to the parent PR, re-opening.

File: source/_components/toon.markdown (hunk 3 above)
Comment (frenck) [reply source/_components/toon.markdown:24]: Incorrect. I agree it feels like it should be a `,`, nevertheless, the `and` delimits two different sum up sections.

File: source/_components/toon.markdown (hunk 2 above)
Comment (frenck) [reply source/_components/toon.markdown:38]: Not sure if that is relevant for our documentation?

File: source/_components/toon.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/toon.markdown:83]: Not sure if we need to place the 30 seconds interval in a note here.

You can change temperatures by calling operations modes. Setting a specific temperature, however, fails. This is caused by a larger issue on the HA platform, which is currently under discussion as an architectural issue. Adding this note here, would require adding it in a lot more places? Not sure about that.

File: source/_components/toon.markdown (hunk 2 above)
Comment (klaasnicolaas) [reply source/_components/toon.markdown:38]: This should actually be clear enough without extra info.

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged 😉 🎉

**Comment by klaasnicolaas** [comment]
Thnx @frenck for this great rewrite of the Toon component 💖

**Pull request merged**